src/tests/test_x509_path.cpp

```diff
@@ -895,6 +895,42 @@ class Path_Validation_With_OCSP_Tests final : public Test
 
 BOTAN_REGISTER_TEST("x509_path_with_ocsp", Path_Validation_With_OCSP_Tests);
 
+class XMSS_Path_Validation_Tests final : public Test
```

Comment:
Alas one more problem - this fails if XMSS is not available, as when building using the NIST build policy.

Reply:
Added the missing guards in a7b7fa47c.